### PR TITLE
tested code improvements

### PR DESCRIPTION
**Pull request by mschilli87**
test results:

I'm getting `not enough reads for analysis [ 0 ]` for most genes when running `pipeline.py` (7 genes get output), `unit_test.py` runs fine & `more_tests.py` returns 25 numbers around 75-80.

### CODE REVIEW

File: estimate_length.py [hunk 1]
```diff
@@ -205,22 +205,26 @@ def extract_pAi_from_genome(genome, window, occurences, consecutive):
             while c <= len(line)-window:
                 segment = line[c:(c+window)]
                 if consecutive*'A' in segment:
```
Comment (mschilli87) [inline estimate_length.py:207]: not sure if I get the sytax here: `consecutive*'A' in segment` is `consecutive * ('A' in segment)` which is `consecutive and ('A' in segment)`? If so, I'd prefer the latter for clarity unless it has some drawbacks I'm not aware of.

**Comment by nukappa** [comment]
sounds good. most of the genes have 0 reads because test data is on chr9.
the numbers from `more_tests.py` are probs, will tell you tomorrow.
is there a `coverage.txt` and a `tail_lengths.txt` generated?
if so and everything looks ok you can merge.

File: estimate_length.py (hunk 1 above)
Comment (nukappa) [inline estimate_length.py:207]: no, if for instance `consecutive=5` then `consecutive*'A' = 'AAAAA'`

Comment (mschilli87) [inline estimate_length.py:207]: ah, ok, didn't get the meaning of `consecutive` then, maybe renaming it to `n_consecutive` and/or switching the factors (`if 'A'*n_consecutive in segment:`) would help but it's clear to me now.

**Comment by mschilli87** [comment]
Just one more thing: `more_tests.py` now depends on `mathplotlib` even though all lines using it are commented out. As we're gonna add plot code eventually I guess that's fine for now.

**Comment by nukappa** [comment]
yes you're right. actually the name `more_tests` is misleading since i keep
unit tests clean and mess up with more_tests to test new code :P

**Pull request merged**